os/drivers/wireless/Kconfig

```diff
@@ -1,16 +1,14 @@
 #
-# For a description of the syntax of this configuration file,
-# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#For a description of the syntax of this configuration file,
+#see kconfig-language at https:	//www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-if DRIVERS_WIRELESS && SELECT_SCSC_WLAN
-
-source drivers/wireless/scsc/Kconfig
-
-endif # DRIVERS_WIRELESS
-if DRIVERS_WIRELESS && SELECT_RTK_WLAN
-
-source drivers/wireless/realtek/Kconfig
-
-endif # DRIVERS_WIRELESS
-
+if DRIVERS_WIRELESS
+&& SELECT_SCSC_WLAN source drivers / wireless / scsc / Kconfig endif
+#DRIVERS_WIRELESS
+if DRIVERS_WIRELESS
+	&& SELECT_RTK_WLAN source drivers / wireless / realtek / Kconfig endif
+#DRIVERS_WIRELESS
+	if DRIVERS_WIRELESS
+		&& BLUETOOTH source drivers / wireless / bluetooth / Kconfig endif
```

Comment:
Please do not change other parts except bluetooth.
Below configure is fine.
--------------------------------------------
if DRIVERS_WIRELESS && BLUETOOTH

source drivers/wireless/bluetooth/Kconfig

endif # DRIVERS_WIRELESS